README.md

```diff
@@ -35,6 +35,26 @@ From there, you can use one of our technology integrations such those for
 Have fun! We have a Google Group and a Slack channel we use for both developers and 
 users. Check them out [here](https://projectnessie.org/develop/).
 
+### Authentication
+
+By default, Nessie servers run with authentication disabled and all requests are processed under the "anonymous"
+user identity.
+
+Nessie supports bearer tokens and uses [OpenID Connect](https://openid.net/connect/) for validating them.
+
+Authentication can be enabled by setting the following Quarkus properties:
+* `nessie.server.authentication.enabled=true`
+* `quarkus.oidc.auth-server-url=<OpenID Server URL>`
```

Comment:
as a follow up PR can we have a little blog showing a full keycloak or google example?

Reply:
I was thinking about a section in the docs, but a blog post is probably even better.